libs/application/templates/accident-notification/src/lib/messages/externalData.ts

```diff
@@ -2,84 +2,105 @@ import { defineMessages } from 'react-intl'
 
 // External information retrieval
 export const externalData = {
-  general: defineMessages({
+  agreementDescription: defineMessages({
     sectionTitle: {
-      id:
-        'an.application:section.backgroundInformation.externalData.sectionTitle',
+      id: 'an.application:section.agreementDescription.sectionTitle',
+      defaultMessage: 'Upplýsingar um meðferð á gögnum',
+      description: 'Information on data handling',
+    },
+    bulletOne: {
+      id: 'an.application:section.agreementDescription.BulletOne',
+      defaultMessage:
+        'Vinnueftirlit ríkisins fær afrit þessa eyðublaðs á grundvelli 4. mgr. 79. gr. laga nr. 46/1980. Tryggingastofnun ríkisins fær upplýsingar um bótagreiðslur sem ekki greiðast samhliða bótum stofnunarinnar, sbr. 14. gr. laga nr. 45/2015..',
```

Comment:
This is just the default message so it maybe doesn't matter too much but there are two dots at the end here

Reply:
Ill fix it in the next pr, was too quick to add automerge 😅